app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.find(1)
```

Comment:
- [ ] You have an easier way to do this, I would suggest you update this to User.first